spec/rspec/mocks/any_instance_spec.rb

```diff
@@ -637,7 +637,7 @@ def inspect
                 expect(klass).to receive(:woot)
                 klass.new.foo
                 verify_all
-              end.to(raise_error(RSpec::Mocks::MockExpectationError) do |error|
+              end.to(fail_with do |error|
```

Comment:
It seems odd to not pass any args to `fail_with`, given that the "with" is intended to read together with the arguments.  Instead, I think this would read better as `fail` w/ a block.